eks/cluster-loader/cluster-loader.gotmpl

```diff
@@ -0,0 +1,53 @@
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: cl-service
+automountServiceAccountToken: true
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: cluster-admin
+subjects:
+  - kind: ServiceAccount
+    name: cl-service
+    namespace: default
+
+---
+apiVersion: batch/v1
+kind: Job
+metadata:
+  name: cl2
+spec:
+  template:
+    spec:
+      containers:
+      - name: clusterloader2
+        image: 197575167141.dkr.ecr.us-west-2.amazonaws.com/clusterloader2:latest
+        command: ["/clusterloader"]
+        args: {{ .TestParams }}
+        volumeMounts:
+        - name: config
+          mountPath: /etc/config
+      restartPolicy: Never
+      serviceAccountName: cl-service
+      volumes:
+        - name: config
+          configMap:
+            name: clusterloader2-config
+
+---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ .ConfigMap.Metadata.Name }}
```

Comment:
If you're gonna make the name of this configurable (I wouldn't), you also need to set this value in the volume of the job.

Reply:
+1